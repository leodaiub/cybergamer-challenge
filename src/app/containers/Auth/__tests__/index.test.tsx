import React from 'react';
import { render } from '@testing-library/react';
import { Store } from '@reduxjs/toolkit';
import { Provider } from 'react-redux';
import { HelmetProvider } from 'react-helmet-async';

import { configureAppStore } from 'store/configureStore';
import { Auth } from '..';

const renderComponent = (store: Store) =>
  render(
    <Provider store={store}>
      <HelmetProvider>
        <Auth />
      </HelmetProvider>
    </Provider>,
  );

describe('<Auth />', () => {
  let store: ReturnType<typeof configureAppStore>;

  beforeEach(() => {
    store = configureAppStore();
  });
  it('should match the snapshot', () => {
    const component = renderComponent(store);
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
