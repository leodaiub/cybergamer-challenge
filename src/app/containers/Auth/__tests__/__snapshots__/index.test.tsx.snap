// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth /> should match the snapshot 1`] = `
.c1 {
  cursor: pointer;
  padding: 1rem;
  background: #ffd700;
  border: 0;
  box-shadow: 0px 2px 4px rgba(0,0,0,0.15);
  border-radius: 6px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 330px;
  height: 54px;
  font-family: 'Ubuntu',sans-serif;
  font-style: italic;
  font-weight: bold;
  font-size: 16px;
  line-height: 18px;
}

.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  class="c0"
>
  <img
    alt=""
    src="logo.svg"
  />
  <div>
    <button
      class="c1"
      type="button"
    >
      ENTRAR
      <img
        alt="button"
        src="arrow_right.png"
      />
    </button>
  </div>
</div>
`;
